runtimes/peregrine/src/lib.rs

```diff
@@ -115,6 +115,22 @@ parameter_types! {
 	pub const SS58Prefix: u8 = 38;
 }
 
+pub struct MigrationFilter;
+impl frame_support::traits::Contains<RuntimeCall> for MigrationFilter {
+	fn contains(c: &RuntimeCall) -> bool {
+		match c {
+			// Enable DidLookup migration calls for ongoing migration
+			RuntimeCall::DidLookup(pallet_did_lookup::Call::migrate { .. }) => {
+				DidLookup::migration_state().is_in_progress()
+			}
+			// For all other DidLookup calls, check whether migration is ongoing
+			RuntimeCall::DidLookup(_) => !DidLookup::migration_state().is_done(),
```

Comment:
Should this be the other way round? All the other calls should be allowed only once the migration is done.

Reply:
I always get these filters wrong. 🙈 07354175